examples/01_template/src/pages/_layout.tsx

```diff
@@ -40,5 +40,5 @@ const getData = async () => {
 export const getConfig = async () => {
   return {
     render: 'static',
-  };
+  } as const;
```

Comment:
this is good. maybe we need this change in all examples with fs-router (incl. managed) with TypeScript.

Reply:
I can do that! do you also want me to generate the entries file in all fs router examples?

Reply:
tbh, i don't think we need to commit entries.gen.tsx files at all with this PR.
it was good for the review purpose.

Reply:
should I add it to the top level gitignore then?

Reply:
Yeah, let's do that.